test/itowns-testing.js

```diff
@@ -102,6 +102,10 @@ global.requestAnimationFrame = (cb) => {
     }
 };
 
+// Fix: process.env.PWD is undefined for Windows
```

Comment:
I may be missing something but wouldn't the proper fix have been to replace the usage of `process.env.PWD` and use `process.cwd()` instead?

```js
const itowns = require(`${process.cwd()}/lib/Main.js`)
```

or is there a good reason to use `process.env.PWD` here?